articles/node_lifecycle.md

```diff
@@ -196,51 +199,176 @@ Transitions to `ErrorProcessing` may be caused by error return codes in callback
 
 - If the `onShutdown` callback raises or results in any other result code the node will transition to `Finalized`.
 
+## Use of a supervisory process
+
+Managed nodes may be instantiated manually by a developer.
+In this case, the developer's own code is responsible for creating and destroying the node.
+However, the node may alternatively be owned by a container system.
+The container will be responsible for creating and destroying the node.
+Such a container will be responsible for exposing the following two transitions.
+
+### Create Transition
+
+This transition will instantiate the node, but will not run any code beyond the constructor.
+
 ### Destroy Transition
 
 This transition will simply cause the deallocation of the node.
 In an object oriented environment it may just involve invoking the destructor.
 Otherwise it will invoke a standard deallocation method.
 This transition should always succeed.
 
-### Create Transition
+## Management Interface
 
-This transition will instantiate the node, but will not run any code beyond the constructor.
+A node that has a managed life cycle complying with the above life cycle shall provide the following interface via ROS topics and services.
+This interface is to be used by tools to manage the node's life cycle state transitions, either automatically or manually according to the tool's purpose.
 
-## Management Interface
+The topics and services of this interface shall function in all states of the node's life cycle.
+They shall not be disabled by the node shifting to the `Inactive` state, for example.
 
-A managed node will be exposed to the ROS ecosystem by the following interface, as seen by tools that perform the managing.
-This interface should not be subject to the restrictions on communications imposed by the lifecycle states.
+### Interface namespace
 
-It is expected that a common pattern will be to have a container class which loads a managed node implementation from a library and through a plugin architecture automatically exposes the required management interface via methods and the container is not subject to the lifecycle management.
-However, it is fully valid to consider any implementation which provides this interface and follows the lifecycle policies a managed node.
-Conversely, any object that provides these services but does not behave in the way defined in the life cycle state machine is malformed.
+The interface shall be provided in a namespace named "infra/lifecycle" underneath the node's namespace.
```

Comment:
It should probably make use of the leading `_` in topic names to make it "hidden":

http://design.ros2.org/articles/topic_and_service_names.html#hidden-topic-or-service-names

Otherwise, it uses a namespace as currently written, so I'm not sure what else you might be referring to.